ansible/tasks/logging/install_fluentd.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Create temporary directory on windows host
+  win_tempfile:
+    state: directory
+  register: tempdir_win
+- name: Fetch ruby devkit exe
+  win_get_url:
+    url: "{{ ruby_fetch_url }}"
+    dest: "{{ tempdir_win.path }}\\ruby_devkit.exe"
```

Comment:
What is the reason to rename the exe? Can't the variable from `main.yml` just be reused here. It will help if we have to log into nodes to debug to have the version in the filename.

Reply:
I will register the variable here instead.